src/components/fx/hover.js

```diff
@@ -670,6 +670,10 @@ function _hover(gd, evt, subplot, noHoverEvent) {
     for(itemnum = 0; itemnum < hoverData.length; itemnum++) {
         var pt = hoverData[itemnum];
         var eventData = helpers.makeEventData(pt, pt.trace, pt.cd);
+
+        // Add axis information to eventData
+        eventData.xaxis = gd._fullLayout[pt.trace.xaxis + 'axis'];
+        eventData.yaxis = gd._fullLayout[pt.trace.yaxis + 'axis'];
```

Comment:
Should I explicitly select the axis' attributes we care about and create a smaller object to include in `eventData`? As far as I know, only `title` was requested.

Reply:
@nicolaskruchten If we decide to be more explicit and only require the attributes that are important, what else beside `(x|y)axis.title` would you like to have available in `hovertemplate`?

Reply:
Hmm aren't `xaxis` and `yaxis` already part of the event data:

![image](https://user-images.githubusercontent.com/6675409/48370667-f6c6d780-e687-11e8-8b8a-ca9683894892.png)

?

Reply:
You are absolutely right: it's already there :tada:! I removed the extra code in 8d95f05 and still passes the test.